src/transformers/generation_utils.py

```diff
@@ -266,7 +266,7 @@ class BeamSampleDecoderOnlyOutput(ModelOutput):
         scores (:obj:`tuple(torch.FloatTensor)` `optional`, returned when ``output_scores=True`` is passed or when ``config.output_scores=True``):
             Processed beam scores for each vocabulary token at each generation step. Beam scores consisting of log
             softmax scores for each vocabulary token and sum of log softmax of previously generated tokens in this beam
-            . :obj:`(max_length,)`-shaped tuple of :obj:`torch.FloatTensor` with each tensor of shape
+            . :obj:`(max_length-1,)`-shaped tuple of :obj:`torch.FloatTensor` with each tensor of shape
```

Comment:
For the "Decoder-only" outputs the length of scores should actually be equal to `max_length - input_sequence_length` whereas `input_sequence_length = input_ids.shape[-1]`. Could you update this maybe for all "DecoderOnly" outputs? :-)

Reply:
I changed the `scores` from the "DecoderOnly" outputs to be `max_length - input_ids.shape[-1]`. Or is it better to change that to be exactly what you suggested (use `input_sequence_length`)?